src/rest_framework_jwt/serializers.py

```diff
@@ -116,6 +116,11 @@ def validate(self, data):
         new_payload = JSONWebTokenAuthentication.jwt_create_payload(user)
         new_payload['orig_iat'] = orig_iat
 
+        # Track the token ID of the original token, if it exists
+        orig_jti = payload.get('orig_jti')
```

Comment:
If someone upgrades a running service to a version including this change, the old, still-valid tokens won't have this claim.

It may be useful later to add a setting that requires that all tokens have a `jti` and all refreshable tokens have a `orig_jti`.

Reply:
Added in a3721e5.

Reply:
I agree. Our primary concern is preserving backwards-compatibility, i.e. that not block existing tokens. Considering this was added later, it's perfectly understandable to ensure a graceful transition for our users using a setting.

If necessary, we can deprecate it later and make it obligatory.